plugins/repository-s3/build.gradle

```diff
@@ -288,7 +288,9 @@ Map<String, Object> expansions = [
   'permanent_bucket': s3PermanentBucket,
   'permanent_base_path': s3PermanentBasePath,
   'temporary_bucket': s3TemporaryBucket,
-  'temporary_base_path': s3TemporaryBasePath
+  'temporary_base_path': s3TemporaryBasePath,
+  'ec2_bucket': s3TemporaryBucket,
```

Comment:
Did you mean for this to be the same as the entry for `temporary_bucket`, and similarly for `ec2_base_path`? I think they should be different.